src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.spec.ts

```diff
@@ -94,6 +103,51 @@ describe('DotEditContentHtmlService', () => {
         })
     );
 
+    it(
+        'should set same height to containers',
+        fakeAsync((): void => {
+            const mockLayout = mockDotLayout;
+            mockLayout.body.rows = [
+                {
+                    columns: [
+                        {
+                            containers: [
+                                {
+                                    identifier: '123',
+                                    uuid: '456'
+                                }
+                            ],
+                            leftOffset: 1,
+                            width: 8
+                        },
+                        {
+                            containers: [
+                                {
+                                    identifier: '321',
+                                    uuid: '654'
+                                }
+                            ],
+                            leftOffset: 1,
+                            width: 8
+                        }
+                    ]
+                }
+            ];
+
+            const querySelector1 = [`div[data-dot-object="container"]`, `[data-dot-identifier="123"]`, `[data-dot-uuid="456"]`].join('');
+
+            const querySelector2 = [`div[data-dot-object="container"]`, `[data-dot-identifier="321"]`, `[data-dot-uuid="654"]`].join('');
+
+            this.dotEditContentHtmlService.setContaintersSameHeight(mockDotLayout);
+            tick();
+
+            const firstContainer = this.dotEditContentHtmlService.getEditPageDocument().querySelector(querySelector1);
```

Comment:
I know I say this but... is not code crying because this method ```getEditPageDocument()``` is private?

Reply:
nope

Reply:
It should... but is the way ```this.dotEditContentHtmlService``` is created, but is ok for now.